src/constants.ts

```diff
@@ -1,2 +1,9 @@
 export const CARD_IAB_CONFIG =
   'directories=no,titlebar=no,toolbar=no,location=no,status=no,menubar=no,scrollbars=no,resizable=no,hidden=yes,clearcache=yes,hidespinner=yes,disallowoverscroll=yes,zoom=no,transitionstyle=crossdissolve';
+
+// new ky first
+export const ENC_KYS = [
```

Comment:
for security, the keys should be #private var
src/providers/key-encrypt/key-encrypt.ts 
 and should not be available outside that class.